arduino/cores/packagemanager/package_manager_test.go

```diff
@@ -744,6 +744,7 @@ func TestFindToolsRequiredFromPlatformRelease(t *testing.T) {
 func TestFindPlatformReleaseDependencies(t *testing.T) {
 	pmb := NewBuilder(nil, nil, nil, nil, "test")
 	pmb.LoadPackageIndexFromFile(paths.New("testdata", "package_tooltest_index.json"))
+	pmb.calculateCompatibleReleases()
```

Comment:
shouldn't this be called automatically inside `pmb.Build()`?

Reply:
Nope, it's called by the commit function returned by the `NewBuilder` method receiver.